tencentcloud/resource_tc_kubernetes_cluster.go

```diff
@@ -1,6 +1,8 @@
 /*
 Provide a resource to create a kubernetes cluster.
 
+~> **NOTE:** To use the custom Kubernetes component startup parameter function, you need to submit a ticket for application.
```

Comment:
**NOTE:** To use the custom Kubernetes component startup parameter function（parameter `extra_args`）, you need to submit a ticket for application.